entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Could/should this go in the `docker_overlay` directory directory or does it make any difference for layer caching?

Reply:
6 of one, half a dozen of the other. I moved it to `docker_overlay` for consistency's sake.